server/src/test/java/io/spine/server/event/EventBusShould.java

```diff
@@ -347,6 +354,43 @@ public void create_validator_once() {
         assertSame(validator, eventBus.getValidator());
     }
 
+    @SuppressWarnings("MethodWithMultipleLoops") // OK for such test case.
+    @Ignore // This test is used only to diagnose EventBus malfunctions in concurrent environment.
+            // It's too long to execute this test per each build, so we leave it as is for now.
+            // Please see build log to find out if there were some errors during the test execution.
+    @Test
+    public void store_filters_regarding_possible_concurrent_modifications()
+            throws InterruptedException {
+        final int threadCount = 50;
+
+        // "Random" more or less valid Event.
```

Comment:
Please have a single text explaining the test scenario. ATM it's totally impossible to understand the idea.